docs/application/native/guides/text-input/autofill.md

```diff
@@ -0,0 +1,374 @@
+# Autofill
+
+Autofill is a feature that allows you to fill commonly entered information, such as email, account, and address in a text input field.
+The autofill functions is used to retrieve previously entered data by the user or request to store data in an autofill service.
+
+The main features of the Autofill API include:
+
+- Autofill Data and Autofill Authentication Information
+
+  You can request to [get the autofill service information](#autofill-data-and-autofill-authentication-information), such as autofill service name, service message, and service logo image using the`autofill_auth_info_request()` function. You can get the existing Autofill data for each view in the Autofill service information received.
+
+- Data Entered and Stored in Autofill Service
+
+  You can [retrieve the data](#data-entered-and-stored-in-autofill-service) (that was stored) in the Autofill service.
+
+- Save Autofill Data
+
+  You can [store](#save-autofill-data) the data entered by the user in an Autofill service.
+
+## Prerequisites
+
+1. To configure the autofill service:
+
+   Autofill service must be configured, to configure, set autofill service by navigating to Settings > Language and input > Input Assistance > Autofill setting.
+   In some other environments, the Autofill service can be found within the system configuration.
+
+2. To use the Autofill API (in [mobile](../../api/mobile/latest/group__CAPI__UIX__AUTOFILL__MODULE.html) and [wearable](../../api/wearable/latest/group__CAPI__UIX__AUTOFILL__MODULE.html) applications), include the `<autofill.h>` header file in your application:
+
+   ```c
+   #include <autofill.h>
+   ```
+
+## Initialization
+
+To use the autofill library,
```

Comment:
Please add colon at the end of the sentence

Reply:
fixed.